source/administration/batch-framework-job-expire.rst

```diff
@@ -0,0 +1,185 @@
+.. _minio-batch-framework-expire-job:
+
+=================
+Batch Expiration
+=================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. versionadded:: MinIO RELEASE.2023-12-02T10-51-33Z
+
+The MinIO Batch Framework allows you to create, manage, monitor, and execute jobs using a YAML-formatted job definition file (a "batch file").
+The batch jobs run directly on the MinIO deployment to take advantage of the server-side processing power without constraints of the local machine where you run the :ref:`MinIO Client <minio-client>`.
+
+The ``expire`` batch job applies :ref:`minio-lifecycle-management-create-expiry-rule` behavior to a single bucket.
+The job determines expiration eligibility based on the provided configuration, independent of any configured expiration rules.
+
+
+Behavior
+--------
+
+Immediate Expiration of Objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Batch expiration occurs immediately as part of the batch job, as compared to the :ref:`passive scanner-based application of expiration rules <minio-lifecycle-management-scanner>`.
+Specifically, batch expiration does not yield to application I/O and may impact performance of regular read/write operations on the deployment.
+
+Expiration Eligibility Determined at Batch-Run
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The batch expiration works per-bucket and runs once to completion.
+The job determines expiration eligibility at the time the job runs, and does *not* rescan or recheck for new objects periodically.
+
+To capture any new objects eligible for expiration, re-run the batch job.
+
+Expiry Rules Check Latest Object Only
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The batch expiration job only checks the latest or "current" version of each object against each batch expiration rule.
+
+
+.. _minio-batch-framework-expire-job-ref:
+
+Expire Batch Job Reference
+--------------------------
+
+.. list-table::
+   :widths: 25 75
+   :width: 100%
+
+   * - Field
+     - Description
+
+   * - ``expire``
+     - *Required* 
+       
+       Top-level field for the expiration job type.
+
+   * - ``apiVersion``
+     - *Required*
+        
+       Set to ``v1``.
+
+   * - ``bucket``
+     - *Required*
+       
+       Specify the name of the bucket in which the job runs.
+
+   * - ``prefix``
+     - *Optional*
+      
+       Specify the bucket prefix in which the job runs.
+
+   * - ``rules``
+     - *Required*
+       
+       An array of one or more expiration rules to apply to objects in the specified ``bucket`` and ``prefix`` (if any).
+
+   * - ``rules.[n].type``
+     - *Required*
+
+       Supports one of the following two values:
+
+       - ``object`` - apply the expiration to an object and its versions, *ignoring* deleted objects (``DeleteMarker`` is current version).
+       - ``deleted`` - apply the expiration to an object and its versions, including deleted objects (``DeleteMarker`` is current version).
```

Comment:
`deleted` rule type applies only to objects with delete marker as their current version. i.e an object that won't be visible when listed using ListObjects API.